ui/src/literature/components/AssignLiteratureItemDrawer.tsx

```diff
@@ -83,6 +64,17 @@ function AssignLiteratureItemDrawer({
     });
   }, [currentUserRecordId, selectedAuthorId, onAssign, paperId]);
 
+  const onAuthorSearch = (value: string): void => {
+    const filteredAuthors = authors.filter(
+        (author: IAuthorResult) => author
+          .get('full_name')
+          .toString()
+          .toLowerCase()
+          .includes(value.toLowerCase())
+      );
+    setAvailableAuthors(filteredAuthors);
+  };
```

Comment:
what if it's an empty search?

Reply:
It will return all of the authors (the same behaviour as if we make an empty search in collections). 
+ I noticed a bug that the pagination wasn't working after search, fixed it :)